woocommerce/woocommerce
# Add reactified main payments screen

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/49927

Add base reactified main payments screen with a demo component



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Turn on feature flag `reactify-classic-payments-settings`

### Test main component
1. In a fresh site with beta tester plugin installed, go through onboarding wizard and set country to Mexico
4. Go to `WooCommerce > Settings > Payments`
5. Observe the `Main payments screen` text as per screenshot
6. Observe notices are still displayed above it
7. Observe breadcrumbs are not shown between the center component and notices

<img width="1316" alt="image" src="https://github.com/user-attachments/assets/eea5e3a8-3bd1-41d1-a223-864430f206d4">

### Test payment gateways
1. Download and activate the following plugins:
    - [woocommerce-payments](https://wordpress.org/plugins/woocommerce-payments/)
    - [klarna-payments-for-woocommerce](https://wordpress.org/plugins/klarna-payments-for-woocommerce/)
    - [woocommerce-paypal-payments](https://wordpress.org/plugins/woocommerce-paypal-payments/)
    - [woocommerce-gateway-stripe](https://wordpress.org/plugins/woocommerce-gateway-stripe/)
1. Go to `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=klarna_payments` and observe it shows Klarna settings
1. Go to `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe` and observe it shows Stripe settings
1. Go to `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=ppcp-gateway` and observe it shows PayPal settings
1. Go to `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments` and observe it redirects to WooPayments screen

### Test offline methods
1. Go to `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=cod` and observe it shows cod form
1. Go to `/wp-admin/admin.php?page=wc-settings&tab=checkout&section= bacs` and observe it shows bacs form
1. Go to `/wp-admin/admin.php?page=wc-settings&tab=checkout&section= cheque` and observe it cheque form



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
